shared/chat/conversation/attachment-video-fullscreen/index.native.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react'
+import {WebView, Box, ClickableBox, Text} from '../../../common-adapters'
+import {globalColors, platformStyles, globalMargins, globalStyles} from '../../../styles'
+import {FloatingMenuParentHOC, type FloatingMenuParentProps} from '../../../common-adapters/floating-menu'
+import type {Props} from './index.types'
+
+class _VideoFullscreen extends React.Component<Props & FloatingMenuParentProps, {loaded: boolean}> {
+  state = {loaded: false}
+  _setLoaded = () => this.setState({loaded: true})
```

Comment:
why bother with this state?

Reply:
I just left it in there by mistake, I'll remove it.